seedlet/assets/sass/structure/_responsive-logic.scss

```diff
@@ -5,9 +5,7 @@
 /**
  * Required Variables
  */
-$flexwidth: 100%;
-$horizontal_space: var(--global--spacing-horizontal);
-
+$default_width: 750px;
```

Comment:
The [figma design comps](https://www.figma.com/file/eBaFrMSiiJxEeUU6T5LU2I/Seedlet?node-id=1166%3A111&viewport=276%2C-2007%2C0.24735313653945923) suggest the default width is 620px.

Reply:
I got 750px from [functions.php](https://github.com/Automattic/themes/blob/d075ea493ecde960752620dc2a3a58221e31f775/seedlet/functions.php#L357). Should we update it there too?